### PR TITLE
Add documentation to promote beta promotion candidates for 2021.11

### PR DESCRIPTION
**Pull request by nicolethoen**

### CODE REVIEW

File: beta-component-promotion/promotion-2021.11/beta-promotion-candidates.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# PatternFly Beta Component Promotion Candidates
+## For Release 2021.11
+
+### Action List
```
Comment (evwilkin) [inline beta-component-promotion/promotion-2021.11/beta-promotion-candidates.md:4]:
```suggestion
### ActionList
```

File: beta-component-promotion/promotion-2021.11/beta-promotion-candidates.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+![action-list](./img/action-list.png)
+
+### Drawer
+A drawer is a sliding panel that enters from the right edge of the viewport. It can be configured to either overlay content on a page or create a sidebar by pushing that content to the left.
+This implementation includes resizable drawers, inline drawers, static drawers, and drawers opening from varying directions.
+
+![drawer](./img/drawer.png)
+
+### Inline Edit
```
Comment (evwilkin) [inline beta-component-promotion/promotion-2021.11/beta-promotion-candidates.md:15]:
```suggestion
### InlineEdit
```

File: beta-component-promotion/promotion-2021.11/beta-promotion-candidates.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+An inline edit component allows users to switch between a read-only view and an edit view of description lists, page text elements, or tables as needed. Inline editing saves the user from opening a modal window or redirecting to a separate edit page so that they can edit in context of their current view.
+This implementation is only done in core and is consumed by other components, such as the inline edit variant of the table.
+
+![inline-edit](./img/inline-edit.png)
+
+### Label Group
```
Comment (evwilkin) [inline beta-component-promotion/promotion-2021.11/beta-promotion-candidates.md:21]:
```suggestion
### LabelGroup
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
LGTM other than a few nits with making component name spacing consistent

**Comment by mcarrano** [comment]
@nicolethoen I think we talked about holding on Tile since we are rethinking the selected states there. Otherwise this looks good. Can you remind me what this page is for? Is this just to share or is this something that is actually being added to the website?

**Comment by nicolethoen** [comment]
@mcarrano good thought. I'll make not of Tile on our tracking and remove it from this.
It's just for reference and sharing - not going on the website - like release highlights.

**Comment by nicolethoen** [comment]
@evwilkin @mcarrano any other thoughts before I merge this? :)

**Review by evwilkin — APPROVED** [review]
LGTM - thanks!

**Review by mcarrano — APPROVED** [review]
Looks good to me, also!

**Pull request merged**